src/main/java/jenkins/plugins/slack/ActiveNotifier.java

```diff
@@ -1,71 +1,57 @@
 package jenkins.plugins.slack;
 
 import hudson.Util;
-import hudson.model.AbstractBuild;
-import hudson.model.AbstractProject;
-import hudson.model.BuildListener;
-import hudson.model.Cause;
-import hudson.model.CauseAction;
-import hudson.model.Result;
-import hudson.model.Run;
+import hudson.model.*;
 import hudson.scm.ChangeLogSet;
 import hudson.scm.ChangeLogSet.AffectedFile;
 import hudson.scm.ChangeLogSet.Entry;
 import hudson.tasks.junit.TestResultAction;
 import hudson.tasks.test.AbstractTestResultAction;
 import hudson.tasks.test.TestResult;
 import hudson.triggers.SCMTrigger;
-import hudson.util.LogTaskListener;
 import jenkins.model.Jenkins;
+import jenkins.plugins.slack.decisions.Context;
+import jenkins.plugins.slack.decisions.NotificationConditions;
+import jenkins.plugins.slack.logging.BuildAwareLogger;
+import jenkins.plugins.slack.logging.BuildKey;
 import org.apache.commons.collections.CollectionUtils;
 import org.apache.commons.lang.StringUtils;
 import org.jenkinsci.plugins.displayurlapi.DisplayURLProvider;
-import org.jenkinsci.plugins.tokenmacro.MacroEvaluationException;
-import org.jenkinsci.plugins.tokenmacro.TokenMacro;
 
-import java.io.IOException;
 import java.text.MessageFormat;
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Set;
-import java.util.logging.Logger;
+import java.util.*;
```

Comment:
can you revert these * import changes please?

Reply:
Done